cli/cli_app.go

```diff
@@ -370,6 +373,11 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		terragruntOptions.IamRole = terragruntConfig.IamRole
 	}
 
+	// replace default sts duration if set in config
+	if terragruntOptions.StsDuration == 3600 && terragruntConfig.StsDuration != nil {
```

Comment:
Why are you checking for exactly equality to 3600? Do you mean `!=`? Any reason not to always set the param, as long as it's not `nil`?

Reply:
The intent here was to check whether `StsDuration` had been overwritten (different from the default of 3600) in `terragruntOptions` (i.e. via CLI or ENV arguments). If it hasn't been overwritten, and `StsDuration` has been defined in `terragruntConfig`, then we should use the `StsDuration` defined in `terragruntConfig`.

I think this is a similar approach to the `DownloadDir` assignment on line 395:397.

Reply:
Ah, OK. Extracting the 3600 into a constant will help make that clear.